src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -226,10 +226,10 @@
         az deployment create --location WestUS --template-uri https://myresource/azuredeploy.json --parameters @myparameters.json
   - name: Create a deployment from a local template file, using parameters from a JSON string.
     text: |
-        az deployment create --location WestUS --template-file azuredeploy.json --parameters '{ \\
```

Comment:
These quotes actually make the example copy-pastable in bash. Removing them breaks this. Where are you seeing a JSON parse error?

Reply:
I am using Ubuntu in Docker on Windows. I have pasted the screenshots below.

I used example from the documentation here:

https://docs.microsoft.com/en-us/cli/azure/resource?view=azure-cli-latest#az-resource-create

First screenshot shows example just pasted directly from the documentation: 

![example-invalid](https://user-images.githubusercontent.com/11562540/58834835-ec514700-8686-11e9-8b13-67588fae4208.png)

There's json parse error, because \ appears to be part of passed json.

Below I removed \ from within '' and now json is parsed correctly (some other error occurs):

![example-valid](https://user-images.githubusercontent.com/11562540/58834849-f2472800-8686-11e9-8ee5-85696195e1b2.png)

Reply:
+ tried on Ubuntu on Windows (original example):

![ubuntu](https://user-images.githubusercontent.com/11562540/58835615-055af780-8689-11e9-9d32-ea68295ffdbf.png)

Reply:
Sorry I mean the backslashes, not quotes.